### ISSUE TITLE
Login form style

### ISSUE THREAD

**Issue by slax0rr** [ClanWolf/C3-Web#4]
Style the login form to resemble the C3 Java Client login form. Even when the C3Web login form will never be shown in the C3 Java Client, they should be as identical as possible when it comes to style.

**Comment by slax0rr** [comment]
@WarWolfen I would ask for a screenshot of the Login form, and any graphical components that I may require, like i.e. that "finger print scan" animation.

**Comment by WarWolfen** [comment]
![image](https://cloud.githubusercontent.com/assets/16292643/22612556/b1a7bfde-ea71-11e6-8a93-5e542a2614b2.png)

**Comment by WarWolfen** [comment]
I was looking for a specific look and feel here. It is quite hard to describe, although anyone probably knows it. It should look like a military sci fi computer system, like we are used to in different movies. I have been looking for a gui set for ages, but have not quite found what was like my imagination. I will add some links here later today.

**Comment by WarWolfen** [comment]
https://www.shutterstock.com/de/image-vector/abstract-creative-concept-vector-futuristic-blue-221866129?src=lb-33886595

**Comment by slax0rr** [comment]
@WarWolfen do you not want to use the style that is already in use by the C3 Client? do you want to change it to one of the linked examples?

**Comment by WarWolfen** [comment]
I am open to change the style of the client as well as soon as I stumble across "the" perfect theme. I think we should use the same images so that they can be exchanged easily if there would be a re-skin...

**Comment by slax0rr** [comment]
Yes, and that's what I requested. Images of the theme, so I can incorporate it into the web design. Sorry if I was unclear.

Can you supply the following graphics:
* Fingerprint scan - is this a gif? If it is only a static image, I can make the "dark bar" move with CSS animations
* background mesh
* that little green text in the background on the bottom right
* satellite behind the finger scan

About the general design of the web site, do you want it in the same frame graphics? Or should I go full web-page-size?

**Comment by slax0rr** [comment]
Discuss this on Friday?

**Comment by WarWolfen** [comment]
Yes, sure. Peter will also be present on friday to show some things.
Meanwhile you can check:
https://github.com/ClanWolf/C3-Java_Client/tree/master/src/main/resources
